### ISSUE TITLE
Issue when importing stimulus controller (controller.name is starting with .-- since last update)

### ISSUE THREAD

**Issue by navidemad** [excid3/esbuild-rails#20]
Hello, thanks for your amazing tool.
Since the last update, the controller.name is starting with .-- for some reasons.
I had to hotfix until a fix is released.

```
import controllers from "./admin/**/*_controller.js";
controllers.forEach((controller) => {
  application.register(controller.name, controller.module.default);
  var controllerName = controller.name.replaceAll(".--", "");
  var controllerImpl = controller.module.default;
  application.register(controllerName, controllerImpl);
});
```

**Comment by excid3** [comment]
Probably due to replacing glob with fast-glob. Will take a look.

**Comment by excid3** [comment]
Released as 1.0.7 🎉 

Thanks @navidemad
